pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -560,6 +560,9 @@ type ArtifactLocation struct {
 
 	// Raw contains raw artifact location details
 	Raw *RawArtifact `json:"raw,omitempty" protobuf:"bytes,7,opt,name=raw"`
+
+	// OSS contains OSS artifact location details
+	OSS *OSSArtifact `json:"oss,omitempty" protobuf:"bytes,2,opt,name=oss"`
```

Comment:
should be `bytes,8`  not `bytes,2`, fix this might fix the build

Reply:
fixed.